tests/components/deconz/test_deconz_event.py

```diff
@@ -53,7 +61,7 @@ async def test_deconz_events(hass):
     assert "sensor.switch_2" not in gateway.deconz_ids
     assert "sensor.switch_2_battery_level" in gateway.deconz_ids
     assert len(hass.states.async_all()) == 3
-    assert len(gateway.events) == 4
+    assert len(gateway.events) == 5
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)